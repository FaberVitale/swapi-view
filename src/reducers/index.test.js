import { default as reducer } from "./index";

describe("reducers/index", () => {
  it("initial state  matches the snapshot", () => {
    expect(reducer(undefined, {})).toMatchSnapshot();
  });
});
