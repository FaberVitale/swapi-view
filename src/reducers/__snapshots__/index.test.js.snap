// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers/index initial state  matches the snapshot 1`] = `
Object {
  "background": Object {
    "status": 0,
    "url": null,
  },
  "data": Object {
    "films": Object {
      "items": Object {},
      "pages": Object {},
      "wookiee": Object {},
    },
    "people": Object {
      "items": Object {},
      "pages": Object {},
      "wookiee": Object {},
    },
    "planets": Object {
      "items": Object {},
      "pages": Object {},
      "wookiee": Object {},
    },
    "search": Object {},
    "species": Object {
      "items": Object {},
      "pages": Object {},
      "wookiee": Object {},
    },
    "starships": Object {
      "items": Object {},
      "pages": Object {},
      "wookiee": Object {},
    },
    "vehicles": Object {
      "items": Object {},
      "pages": Object {},
      "wookiee": Object {},
    },
  },
  "drawer": false,
  "fetchState": Object {
    "failed": Object {},
    "pending": Array [],
  },
}
`;
